Makefile

```diff
@@ -14,18 +14,20 @@ endif
 
 OPENAPI ?= docker run --rm \
 		--user=$(shell id -u $(USER)):$(shell id -g $(USER)) \
-		-v $(shell pwd):$(shell pwd) \
+		-v ${PWD}:${PWD} \
 		openapitools/openapi-generator-cli:v5.1.1
 
-all: ui api kubernetes filesystem
+all: ui build lint
 
-install:
-	cd ui && npm install
+clean:
+	rm -rf ui/build ui/node_modules cmd/api/ui/build
```

Comment:
Or just go for `git clean -Xdf` all together?

Reply:
Yup, that would work too.